IdentityCore/src/oauth2/token/MSIDAccessToken.m

```diff
@@ -194,7 +194,7 @@ - (void)setResource:(NSString *)resource
 
 - (NSOrderedSet<NSString *> *)scopes
 {
-    return [_target msidScopeSet];
+    return [NSOrderedSet msidOrderedSetFromString:_target normalize:NO];
```

Comment:
Should we change `msidScopeSet` so it will not normalize scopes too? Currently, we use `msidScopeSet` in many places, including broker_v2.

Reply:
scopes are case sensitive, we shouldn't normalize them.

Reply:
That's a tricky situation. Scopes are case sensitive by spec. However, AAD normalizes scopes for us in the result, so AAD treats scopes in a case insensitive manner. Therefore, we must treat scopes in a case insensitive manner for cache lookups. For everything else, we can treat them either case insensitive or sensitive. To make sure we're consistent with other MSALs and more consistent with other IDPs, I made a change to treat scopes in a case sensitive manner for all requests, but still kept them case insensitive for cache lookups.

Reply:
would it be more correct then to 
- normalize for AAD requests, and 
- not normalize for other requests?
rather than doing case insensitive cache look up for AAD only and returning them?

It just seems like that would be easier to code and maintain/fix in case problem arises.

Reply:
Based on recent finding, resource in scope is treated as case sensitive by AAD and scope as case insensitive. I think it makes sense to stick with the original proposal of:
1. Keep case for network requests
2. Keep case for responses
3. Treat scopes in a case insensitive manner to avoid issues when AAD normalizes scopes in response

I updated the original PR to use that logic